src/index.html

```diff
@@ -1,13 +1,20 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Antisnake</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Antisnake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Antisnake</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="box">
+      <div class="block1">1</div>
+      <div class="block2">2</div>
+      <div class="block3">3</div>
+      <div class="block4">4</div>
+      <div class="block5">5</div>
+      <div class="block6">6</div>
```

Comment:
check [checklist](https://github.com/mate-academy/layout_antisnake/blob/master/checklist.md) point 2

Reply:
Good point, fix it